tests/integration/config/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "tokenKey": "49108073e18547b20214d77575d10646",
+    "tokenURL" :"https://api.jwt-fetch.mgmt.smc.dvsacloud.uk/1.0/nonprod/vta",
+    "username":"CVS.FullAccess@dvsatest-cloud.uk",
+    "password":"evIX-*#57.-!Gz"
+}
```

Comment:
Agree with Dan the settings.json is usually just held locally on your machine and not committed to the repo.